DuckDuckGo/AppDelegate.swift

```diff
@@ -82,13 +82,22 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
     
     private func startOnboardingFlowIfNotSeenBefore() {
+        
         var settings = TutorialSettings()
         if !settings.hasSeenOnboarding {
+            startMigration()
```

Comment:
Similar to what you have now - just make a separate method to handle migration logic and call it before the `startOnboardingFlowIfNotSeenBefore()` method rather than within.